### ISSUE TITLE
[GeoNature] Recherche avancée taxon - rang taxonomique à l'espèce

### ISSUE THREAD

**Issue by ch-cbnmc** [cbn-alpin/sinp-aura-tickets#43]
Possibilité de descendre la recherche par rang taxonomique du genre à l'espèce?

**Comment by jpm-cbna** [comment]
Au niveau du filtre avancé de la Synthese (menu Observations) ? Pour quelles raisons ?
Est ce que la recherche par taxon (non avancé) ne fait pas justement le job ?

**Comment by ch-cbnmc** [comment]
Les botanistes voudraient avoir, quand ils cherchent une espèce, les sous espèces qui vont avec - ce qui n'est pas le cas avec la recherche normale (et je crois que côté CBNA ça ne pose pas de questions parce que tout est ramené à l'espèce sur GN?), d'où la possibilité de la recherche avancée, mais qui ne va pas jusqu'à l'espèce.

**Comment by jpm-cbna** [comment]
Ok, je comprends mieux. En attendant, il est toujours possible de lancer une recherche en ajoutant toutes les sous-espèces. Je pense que c'est la raison de l’absence de la fonctionnalité... Mais ok, quand il y en a beaucoup c'est pas pratique.

**Comment by ch-cbnmc** [comment]
Oui, effectivement c'est aussi la réponse que je fais (il est possible d'ajouter à la recherche chaque sous espèce), mais évidemment ils ne trouvent pas ça très pratique, et spontanément quand la recherche est faite à l'espèce ils pensent que les sous-espèces sous inclues...

**Comment by ch-cbnmc** [comment]
Pour info (remarque remontée d'un botaniste), sur le sujet: biodiv Atlas permet ça: exemple une recherche Seseli annuum sur Atlas te précise "2 taxons inférieurs agrégés sur cette fiche" (Séséli annuel | Seseli annuum subsp. annuum et Séséli à feuilles de Carvi | Seseli annuum subsp. carvifolium);
Alors que sur Expert il faut ajouter espèce + chacune des sous espèces possibles pour avoir la carte équivalente.
